loaders/gtdb/gtdb_config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "LOAD_VERSION": 207,
+  "KBASE_COLLECTION": "gtdb",
+  "LOAD_FILES": [
+    "~/SCIENCE/taxonomy/gtdb/bac120_taxonomy_r207.tsv",
+    "~/SCIENCE/taxonomy/gtdb/ar53_taxonomy_r207.tsv"
+  ]
+}
```

Comment:
As an alternative option we discussed, given the relatively small number of inputs, just having CLI params rather than a config. What made you decide to go with a config file?

Reply:
👍 a CLI param should be the way to go.

Reply:
Ok, I'm confused then - if we're going to use CLI params do we need this PR at all?

Reply:
hmmm, we probably still need a global config for DB connections. maybe some default params that go into config files?

Reply:
But if we use arangoimport we don't need DB connection config, right?